src/wallet/rpcwallet.cpp

```diff
@@ -2982,6 +2982,14 @@ static UniValue settxfee(const JSONRPCRequest& request)
     LOCK2(cs_main, pwallet->cs_wallet);
 
     CAmount nAmount = AmountFromValue(request.params[0]);
+    CFeeRate nFeeRate(nAmount, 1000);
```

Comment:
nit: Should be snake_case: `tx_fee_rate`.

Also, could submit this feature independent of this pull request, i.e. create a separate pull request.